airbyte-webapp/src/components/ui/Button/Button.module.scss

```diff
@@ -153,6 +151,10 @@
     &:active {
       background-color: colors.$blue-600;
     }
+
+    &:focus-visible {
+      outline-offset: 1px;
```

Comment:
For primary buttons, give a 1px offset so there is contrast.

Open to _always_ applying the offset, but the designs Nico has, they are hugging the element.